airbyte-integrations/bases/connector-acceptance-test/unit_tests/test_core.py

```diff
@@ -1331,6 +1337,188 @@ def test_validate_field_appears_at_least_once(records, configured_catalog, expec
         t._validate_field_appears_at_least_once(records=records, configured_catalog=configured_catalog)
 
 
+@pytest.mark.parametrize(
+    "output, expected_exception",
+    [
```

Comment:
Can we add a descriptive name for those tests? As a reader of those tests, I should be able to spot the one thing we test very easily.

It would be useful because it feels like some of those tests test multiple things. For example, I would assume only two tests to have more than one stream: one that fails and one that succeeds. For the others, we can simplify them to have only one stream. It feels that we might be using 3 streams every time because the parametrized test assume there are three. This might mean that using `parametrize` here is not optimal and we can split them

Reply:
agreed, moved successful case to another test (async_stream_output)
left `parametrize` for `Exception` cases.